lib/ontologies_linked_data/models/category.rb

```diff
@@ -6,10 +6,10 @@ class Category < LinkedData::Models::Base
       attribute :name, enforce: [:existence]
       attribute :description
       attribute :created, enforce: [:date_time], default: lambda { |record| DateTime.now }
-      attribute :parentCategory, enforce: [:category]
+      attribute :parentCategories, enforce: [:category, :list]
```

Comment:
you can't change the name of the attribute, otherwise, you will lose the current data.

Reply:
We don't have any data currently

Reply:
BioPortal does have https://data.bioontology.org/categories?apikey=8b5b7825-538d-40e0-9e9e-5ab9274a9aeb, and we need to be still backward compatible

Reply:
You are right. Okay